lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -6,6 +6,7 @@ SQLAlchemy==1.0.15
 mercurial==3.7.3
 numpy==1.9.2
 pycrypto==2.6.1
+pyuwsgi==2.1.dev0+gx1.c6ee1f6
```

Comment:
Hrm, yeah, possibly. Since @jxtx's original work in galaxyproject/starforge#95 may have been destroyed I'm not 100% clear.